diff -up dhcpcd-9.4.0/hooks/20-resolv.conf.2~ dhcpcd-9.4.0/hooks/20-resolv.conf
--- dhcpcd-9.4.0/hooks/20-resolv.conf.2~	2020-12-28 20:41:53.048737346 +0100
+++ dhcpcd-9.4.0/hooks/20-resolv.conf	2020-12-28 20:42:51.210707822 +0100
@@ -172,7 +172,8 @@ add_resolv_conf()
 	done
 	if $have_resolvconf; then
 		[ -n "$ifmetric" ] && export IF_METRIC="$ifmetric"
-		printf %s "$conf" | "$resolvconf" -a "$ifname"
+		# resolvconf doesn't support multiple arguments, that's why it's run twice
+		printf %s "$conf" | "$resolvconf" -a "$ifname" && "$resolvconf" -u
 		return $?
 	fi
 
